src/components/panel/panel.js

```diff
@@ -284,6 +284,15 @@ angular
  * toggled the class and animations are completed.
  */
 
+/**
+ * @ngdoc method
+ * @name MdPanelRef#focusOnOpen
+ * @description
+ * Focuses the panel content if the focusOnOpen config value is true. This should
```

Comment:
I wonder if we should just do it on both? Focusing on something with `display: none` or `visibility: hidden` doesn't do anything. The only issue would be `opacity: 0`, but that element is accessible to screen readers anyways, so they can still get to it and interact with it (therefore setting the focus doesn't seem too bad).